tensorflow/lite/kernels/mul_test.cc

```diff
@@ -337,6 +337,25 @@ TEST(QuantizedMulOpTest, NoActivationInt16) {
                                               kQuantizedToleranceInt16)));
 }
 
+TEST(QuantizedMulOpTest, NoActivationInt16Scaled) {
+  const float kMin = -2.f;
+  const float kMax = 2.f*32767.f / 32768.f;;
+  QuantizedMulOpModel m({TensorType_INT16, {1, 2, 3, 1}, kMin, kMax},
```

Comment:
@wwwind here is the internal lint error , can you please fix it 
`CheckLint found errors.
  Line contains only semicolon. If this should be an empty statement, use {} instead.  [whitespace/semicolon] [5]
	//depot/google3/third_party/tensorflow/lite/kernels/mul_test.cc:343`